### ISSUE TITLE
Hoover improvements

### ISSUE THREAD

**Issue by alsegno** [liquidinvestigations/liquidinvestigations#23]
Umbrella issue for future hoover improvements

**Comment by alsegno** [comment]
May include some authentication work (Yubi keys)

**Comment by alsegno** [comment]
Refer here: http://sentre.de:6080/grain/cMhWLtbhRMZQnscXFpBG92

**Comment by alsegno** [comment]
This encompasses #22 and #8

**Comment by germakoci** [comment]
don't forget some mock-ups are done for some of the features, and uploaded:
- some here https://github.com/hoover/design/tree/master/v2 and here https://github.com/hoover/design/tree/master/v1 
- and here for batch search https://github.com/hoover/design/tree/master/v3-batch-search

**Comment by mgax** [comment]
Those issues should happen on the hoover repos.

**Comment by germakoci** [comment]
I agree, sure; but some of these should be part of the liquid project and delivered during the liquid project

**Comment by alsegno** [comment]
per discussion with @germakoci moved to q4, backlog
